### PR TITLE
Inkbird fixes and 6xS merge with Solis 6

### PR DESCRIPTION
**Pull request by DigiH**
• Inkbird leading manufacturerdata fixes
• merge of 6XS with Solis 6
• test corrections
• docs fixes

## Checklist:
  - [x] The pull request is done against the latest development branch
  - [x] Only one feature/fix was added per PR and the code change compiles without warnings
  - [x] I accept the [DCO](https://github.com/theengs/decoder/blob/development/docs/participate/development.md#developer-certificate-of-origin).

### CODE REVIEW

File: src/devices/IBT_6XS_SOLIS6_json.h [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+/*R""""(
+{
+   "brand":"Inkbird/Tenergy",
+   "model":"iBBQ/SOLIS6",
+   "model_id":"IBT-6X(S)/SOLIS-6",
```
Comment (1technophile) [inline src/devices/IBT_6XS_SOLIS6_json.h]: I'm worried about the parenthesis in the model id. Not sure how the different client programs handle this. Maybe we stick to IBT-6XS/SOLIS-6 ?

**Comment by 1technophile** [comment]
Thanks!

File: src/devices/IBT_6XS_SOLIS6_json.h (hunk 1 above)
Comment (DigiH) [reply src/devices/IBT_6XS_SOLIS6_json.h]: Yes, I wasn't actually sure about that, so best to keep that just in the docs with the parenthesis.

Also thinking about renaming the Solis 6 to just Solis, as it is also being sold with just 2 probes, but still has all the 6 connectors for additional probes. What do you think? Or keep it with the 6 as it still has 6 connectors ;)

Comment (DigiH) [reply src/devices/IBT_6XS_SOLIS6_json.h]: Resubmitted with just the parenthesis change.

Comment (1technophile) [reply src/devices/IBT_6XS_SOLIS6_json.h]: Yes, makes sense to rename it to Solis only

**Pull request merged**